spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/ReactorResilience4JCircuitBreaker.java

```diff
@@ -91,4 +99,10 @@ public ReactiveResilience4JCircuitBreaker(String id,
 		return toReturn;
 	}
 
+	@Override
+	public <T> Publisher<T> run(Publisher<T> toRun,
```

Comment:
since this is a duplicate of what we have in Hystrix, perhaps this should be a default interface method? It looks generic enough

Reply:
No :( Naming becomes an issue.  Since `ReactorCircuitBreaker` already has a run method that takes a `Flux` and `Flux` extends `Publisher` it ends up being an ambiguous method call.